packages/services/src/linkResolver.service.ts

```diff
@@ -351,6 +351,15 @@ export const buildElementString = (ai: any) => {
 };
 
 export const extractFromElementString = (elementString: string) => {
+  let preparedElementString = elementString;
```

Comment:
I am curious why we assign this `elementString` variable.

Reply:
Because it will change the input parameter.